connectors/kafka/src/test/java/forklift/integration/ConsumerActionTests.java

```diff
@@ -0,0 +1,42 @@
+package forklift.integration;
+
+import forklift.Forklift;
+import forklift.connectors.ConnectorException;
+import forklift.consumer.Consumer;
+import forklift.exception.StartupException;
+import forklift.producers.ForkliftProducerI;
+import forklift.producers.ProducerException;
+import org.junit.Test;
+
+public class ConsumerActionTests extends BaseIntegrationTest {
+
+    /**
+     * Test subscribing to a topic, unsubscribing, then resubscribing to the topic.
+     */
+    @Test
+    public void topicTurnoverTest() throws StartupException, ConnectorException, InterruptedException, ProducerException {
+
```

Comment:
Your use of newlines like this seems inconsistent in this PR; I personally don't like this style, but :man_shrugging: